jbmc/src/java_bytecode/generic_parameter_specialization_map_keys.h

```diff
@@ -23,12 +23,10 @@ class generic_parameter_specialization_map_keyst
 public:
   /// Initialize a generic-parameter-specialization-map entry owner operating
   /// on a given map. Initially it does not own any map entry.
-  /// \param _generic_parameter_specialization_map: map to operate on.
+  /// \param generic_parameter_specialization_map: map to operate on.
```

Comment:
Probably split commit "Code cleanup", e.g. "factor out get_final_name_component", one commit per variable rename, the stuff with `new_class_type`, the re-spacing are all pretty independent changes.